packages/material-ui/src/InputAdornment/InputAdornment.js

```diff
@@ -16,10 +16,11 @@ export const styles = {
   },
   /* Styles applied to the root element if `variant="filled"`. */
   filled: {
-    '&$positionStart': {
+    '&$positionStart:not($marginDense):not($hiddenLabel)': {
```

Comment:
```suggestion
    '&$positionStart:not($hiddenLabel)': {
```

It would solve this case: 
```jsx
      <TextField
        id="filled-adornment-dense"
        className={clsx(classes.margin, classes.textField)}
        variant="filled"
        margin="dense"
        label="amount in Euro"
        InputProps={{
          startAdornment: <InputAdornment position="start">€</InputAdornment>,
        }}
      />
```

Outputs:
![Capture d’écran 2019-07-08 à 17 20 27](https://user-images.githubusercontent.com/3165635/60817624-b252f380-a1a4-11e9-89c4-6f97f65e165c.png)

Reply:
Nice. Will include this on the docs page as well for regression test.

We should clean those up at some point and just automatically generate some regression test that uses combinations of props.

Reply:
The visual regression tests don't scale very well with the number of screenshots. That might be the first challenge to solve in order to use a combination of props. (My assumption is that a combination of props will generate significantly more diff cases).

Reply:
I would pack those into a single screenshot just like it is now.

Reply:
Done

Reply:
To dive a bit more, I think that we have 3 different potential bottlenecks in the screenshot generation pipeline. 1. The generation of the images, 2. The upload of the images and 3. The diff of the images. An analysis of the time each of these task take-ups will help further evaluate which point is the main bottleneck right now. The upload time seems to be negligible (2.) 2s. Step n°1 takes about 100s. The diff time seems to take about twice as much time.

So we might have a bottleneck first linked to the surface area we screenshot and second with the rendering time on the browser.